### PR TITLE
Column sort order & JS Warning Fix

### PR DESCRIPTION
**Pull request by jmuharsky**

### CODE REVIEW

File: client/components/widget/query/data-view-service.js [hunk 1]
```diff
@@ -51,13 +52,14 @@ goog.provide('p3rf.perfkit.explorer.components.widget.query.DataViewService');
 
 goog.require('p3rf.perfkit.explorer.components.widget.data_viz.gviz.getGvizDataView');
 goog.require('p3rf.perfkit.explorer.models.DataViewModel');
+goog.require('p3rf.perfkit.explorer.models.SortOrder');
 
 goog.scope(function() {
 var explorer = p3rf.perfkit.explorer;
+var SortOrder = explorer.models.SortOrder;
```
Comment (cmccoy) [inline client/components/widget/query/data-view-service.js]: Nit: should these aliases be ordered (so, DataViewModel first?)

File: client/components/widget/query/data-view-service.js [hunk 2]
```diff
@@ -71,6 +73,10 @@ explorer.components.widget.query.DataViewService = function(GvizDataView) {
    * @private
    */
   this.GvizDataView_ = GvizDataView;
+  
+  /**
+   * @type
+   */
```
Comment (cmccoy) [inline client/components/widget/query/data-view-service.js]: Is this right?

File: client/components/widget/query/query-editor-directive.html [hunk 3]
```diff
@@ -126,6 +126,20 @@
...
         </div>
       </div>
+
+      <div ng-show="queryEditorCtrl.datasource.view.sort_columns">
+        <div class="sidebar-item">
+          <div class="sidebar-item-label">sort order</div>
+          <div class="sidebar-item-value">
+            <select class="form-control"
+                   ng-model="queryEditorCtrl.datasource.view.sort_column_order">
+              <option value="" selected>Ascending (default)</option>
```
Comment (cmccoy) [inline client/components/widget/query/query-editor-directive.html]: "sort order" is blank when I view this page, which makes having two "Ascending" options a bit confusing...

Comment (jmuharsky) [inline client/components/widget/query/query-editor-directive.html]: Interesting... it shouldn't be "blank", it should be:

Ascending (default)
Ascending
Descending

I use this pattern in the default table style as well.

File: client/components/widget/query/data-view-service.js (hunk 2 above)
Comment (jmuharsky) [inline client/components/widget/query/data-view-service.js]: Nope; totally extraneous and will be removed.

File: client/components/widget/query/data-view-service.js (hunk 1 above)
Comment (jmuharsky) [inline client/components/widget/query/data-view-service.js]: Done.

**Comment by cmccoy** [comment]
LGTM.

**Pull request merged**